ActivityHistoryApi/V1/Domain/Enums.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+
+namespace ActivityHistoryApi.V1.Domain
+{
+    [JsonConverter(typeof(JsonStringEnumConverter))]
+    public enum TargetType
+    {
+        person,
+        asset,
+        tenure
+    }
+
+    [JsonConverter(typeof(JsonStringEnumConverter))]
+    public enum Type
```

Comment:
I would rename this to be `ActivityType`, otherwise it gets really confusing with the built-in `System.Type` class.